### ISSUE TITLE
NP-6158-0 error on Criminal Girls: Invite Only (PCSE00516)

### ISSUE THREAD

**Issue by kingdwemen** [r0ah/vitacheat#107]
**Describe the issue**
PCSE00516 has a couple psv cheat files in the database, one for a modded and unmodded game- I'm using an unmodded copy and patching it via RePatch. However, trying to load the game with the Vitacheat plugin enabled causes an NP-6158-0 error, and the game won't start. I tried disabling RePatch as the troubleshooting section suggests, but it still won't get past this error. The game launches fine without VitaCheat enabled. Is there another way to load these cheats?

**My info:**
 - **Your PSVita Firmware version**: 3.60Enso
 - **ID**: PCSE00516
 - **Version**: 1.00
 - **Type**: NoNpDrm

**Comment by Yohoki** [comment]
looking at the Reddit post by u/jeff7360:
"To install the mod:

On NoNpDrm or Legit games: drop the gameid folder into ux0:\patch (the data.dat is the FULL data.dat so the same file in the original game folder should be deletable and retain playability. YMMV)"

You shouldn't need to use RePatch for this game. Delete the game and restart.

Load the game into your vita as you normally would for NND games by placing the PCSE00516 folder into your ux0:apps/ folder. Reload database and licenses then check if the game runs. Load the patch into ux0:patch/ folder. You can save some room on your SD card or w/e by deleting the data.dat file from the ux0/apps/PCSE00516 folder.

**Comment by kingdwemen** [comment]
Woop, sorry- hadn't actually checked to remember if I was even using rePatch for this mod. Yeah, I have the files in uxo:/patch, as per the guide. However, I did try removing the .dat from inside the game folder, to see if that did anything. No dice, same crash.

**Comment by Yohoki** [comment]
according to the PSVitaDevWiki, the error code has to do with trophies not loading properly.

~~~~
SCE_NP_TROPHY_ERROR_NOT_INITIALIZED | 0x80551601 | NP-6158-0
~~~~

Looks like you may need to delete your trophy data for this game.
Go to vitashell and navigate to ur:user/00/trophy/conf/
Delete the folder "NPWR07778_00"
Go to ur:user/00/trophy/data/
Delete the folders "NPWR07778_00" and "sce_trop"
Exit Vitashell and open the Trophies app. It will say it's rebuilding the trophies or something. Try loading the game after that.

Don't connect to wifi while rebuilding trophies or it will get the ones stored on sony's servers, which may already include the criminal girl's trophies that aren't working.

**Comment by kingdwemen** [comment]
I remember seeing something about trophies before- did as you said, but still the same crash. 

Apparently I'm not the only one with this issue- 

https://www.nextgenroms.com/topic/6773-ps-vita-error-np-6158-0/

But this was back in 2017, and the codes were just added this last November. Maybe it's the specific version of the plugin I'm using? I downloaded and installed vitacheat through AutoPlugin, it was the "Z05 BETA FOR 3.60" version.

**Comment by Yohoki** [comment]
I didn't use autoplugin for vitacheat. I'm pretty sure I made the codes on z06 Vitacheat, because I grabbed that version not long after it was released. I don't think that would have any effect on it loading, though.

If you want to try upgrading to 3.65 and getting z06 (and rePatch, since that's needed on 3.65) you can, but I've no idea if that will solve the issue. I posted the codes not too long ago, but I had been toying with the game for several months... but I'm pretty sure I was on 3.65 when I made them

You might try hitting up the GBATemp Vitacheat thread to see if the hivemind there can give some more help, because I have no idea why it's not running.
https://gbatemp.net/threads/vitacheat-finalcheat-database.485343

**Comment by kingdwemen** [comment]
Ah, looking through that thread I noticed someone already had brought up their game crashing from the codes to you- though, them having to reboot from safe mode is not the same issue as I'm having. 

I think it's definitely an issue with the 3.60 version of the plugin, maybe it's something that got patched by the Z06 release on 3.65? The 2017 crash from the nextgen thread would make sense, as that would be Z05 and below (assuming Z06 was indeed released only last year.)

The thought of updating the vita firmware just to cheat in one game isn't appealing to me lol. And asking for a back-port of the plugin would bring it's own slew of problems (the possibility of the codes not working on 3.60/having to be remade, for example), so I think I'm willing to let it rest here. Thanks a lot for helping me poke at this, at least I'll have peace of mind knowing I tried everything but the kitchen sink.

**Comment by Yohoki** [comment]
Ya, his was definitely something wasn't going right, but he didn't give me any info about what was up (FW, dump type, list of plugins used, etc.) so I have no idea if it's something installed wrong, or a plugin in the way.

I know that looking up this error code show this game and 3.60 several times on the first page, so it's looking highly likely that 3.60 is the cause. So, upgrading would likely fix the issue for you. It's a hassle, but that's what it's looking like.

And to be real, z06 vitacheat only works on 3.65, and a lot of my newer codes are made using features that only z06 supports (crashes vita with lower versions)... so that's another benefit, that newer codes may not work with your current setup. But, that's all up to you.

I'm going to go ahead and close this issue, since the issue seems to be 3.60+Vitacheat is a bad combo for this game. I'll update the game's psv file with this info.
